jobboard/migrations/0008_merge_20211219_1816.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.2.9 on 2021-12-19 18:16
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('jobboard', '0004_alter_job_work_from'),
+        ('jobboard', '0007_add_example_companies'),
+    ]
+
+    operations = [
```

Comment:
What's the point of this file? Looks like it doesn't do anything or am I missing something?

Reply:
when I added a new migration file for some reason I had a conflict between 004 and 007 files, In this way, I didn't change them and resolved the conflict with the Django migration default option.

Reply:
Ok, looks a bit weird that the operations list is empty, but maybe it's arlight